src/scss/_header-nav.scss

```diff
@@ -0,0 +1,69 @@
+@use './general';
+
+.header-nav {
```

Comment:
I always use prefixes in BEM selectors, like `.cmp-` for components, `.obj-` for objects, and `.util-` for utilities. This helps a lot when inspecting code in DevTools and tracking down the code for whatever I'm debugging. There's also sitemaps, so if that's your jam, go for it 🤷

Reply:
I've seen the `.cmp` and `.obj` with Philip, so I think I can manage. I hadn't moved any of the code from `base.njk` to actual components yet, so I wasn't thinking about those prefixes. But if I know I'm going to do that (at least for things that should be shared components across multiple pages), I can get a jump start. I'll do that in the following PRs too. 

tada! https://github.com/marissahuysentruyt/marissahuysentruyt/pull/21/commits/7f3c868bd72940b204e4145a0e307ded4d803d7c

Do you think I should still keep the `header-nav` portion, or just reduce it to `header`?

Reply:
I think it makes sense to reduce it to `header`, especially now that the "block" is on the `<header>` element itself. Not a big deal either way, though.

Reply:
Once I added the `cmp` prefix, I was thinking it wasn't necessary anymore. Thanks! https://github.com/marissahuysentruyt/marissahuysentruyt/pull/21/commits/cfb38a6418e6c86fe31f7e6757a9164698470871